src/traccuracy/_tracking_graph.py

```diff
@@ -169,18 +170,23 @@ def __init__(
                 "annotation. Please change the frame key."
             )
         self.frame_key = frame_key
-        if label_key is not None and NodeFlag.has_value(label_key):
-            raise ValueError(
-                f"Specified label key {label_key} is reserved for graph"
-                "annotation. Please change the label key."
-            )
-        self.label_key = label_key
-        for loc_key in location_keys:
-            if NodeFlag.has_value(loc_key):
+
+        def _check_valid_key_name(key: str, name: str) -> None:
```

Comment:
Would just pull this function outside of the class, either in the same file or in a `utils` type file.